### PR TITLE
[14.0][MIG] partner_contact_in_several_companies

### PR DESCRIPTION
**Pull request by jacob-willdooit**
Versioning updated, tests are passing.
Minor fixes were required to get the test to pass.

Should be good to go.

### CODE REVIEW

**Review by richard-willdooit — APPROVED** [review]
LGTM.

**Comment by richard-willdooit** [comment]
@njeudy Do you still have an interest in this module and are you able to check it over?

**Review by dsolanki-initos — APPROVED** [review]
Code review and functional test. LGTM

**Comment by richard-willdooit** [comment]
@dsolanki-initos Is there anyone in partticular who needs to trigger the merge?

File: partner_contact_in_several_companies/tests/test_partner_contact_in_several_companies.py [hunk 1]
```diff
@@ -121,7 +121,8 @@ def test_04_contact_creation(self):
         # Reset contact to standalone
         new_contact.write({"contact_id": False})
         self.assertEqual(
```
Comment (NL66278) [inline partner_contact_in_several_companies/tests/test_partner_contact_in_several_companies.py]: Why not make this a oneliner?

File: partner_contact_in_several_companies/tests/test_partner_contact_in_several_companies.py [hunk 2]
```diff
@@ -143,36 +144,35 @@ def test_05_contact_fields_sync(self):
         # Test DOWNSTREAM sync
         self.bob_contact.write({"name": "Rob Egnops"})
         self.assertEqual(
-            self.bob_job1.name, "Rob Egnops",
```
Comment (NL66278) [inline partner_contact_in_several_companies/tests/test_partner_contact_in_several_companies.py]: Same here. In general: if it fits on one line, put in on one line.

File: partner_contact_in_several_companies/tests/test_partner_contact_in_several_companies.py [hunk 3]
```diff
@@ -143,36 +144,35 @@ def test_05_contact_fields_sync(self):
...
+        """Check ir_action context is auto updated."""
 
         new_context_val = (
             "'search_show_all_positions': " "{'is_set': True, 'set_value': False}"
         )
 
-        details = self.env["ir.actions.act_window"].for_xml_id(
-            "base", "action_partner_form"
-        )
+        details = self.env.ref("{}.{}".format("base", "action_partner_form")).read()[0]
```
Comment (NL66278) [inline partner_contact_in_several_companies/tests/test_partner_contact_in_several_companies.py]: Much clearer here also to just specify self.env.ref("base.action_partner_form"). Like you do a few lines below with "partner_contact_in_several_companies.action_partner_form".

**Review by NL66278 — CHANGES_REQUESTED** [review]
:+1: Thanks for your contribution! A few small changes would make it perfect.

**Comment by jacob-willdooit** [comment]
Hi @NL66278 I've made the requested changes.

**Review by NL66278 — APPROVED** [review]
:+1: LGTM. Thanks again!

**Pull request merged**